Nodejs/Product/Nodejs/Extensions.cs

```diff
@@ -195,7 +195,9 @@ out project
 
             if (pyProj != null) {
                 analyzer = pyProj.Analyzer;
-                return analyzer;
+                if (analyzer != null) {
+                    return analyzer;
+                }
```

Comment:
This was needed for the previous logic. Will revert since now only the JsAnalyzer is lazily created instead of the VsProjectAnalyzer.